### ISSUE TITLE
[Filter Bar]: [Missing ARIA Level in "Adapt Filters" Button

### ISSUE THREAD

**Issue by JackieWei** [SAP/ui5-webcomponents-react#6994]
### Describe the bug

The "Adapt Filters" button should include appropriate ARIA attributes (aria-level, aria-role) to provide semantic information about its purpose. The screen reader should announce the button as a functional element with proper context.

### Reproduction steps

1. Add FilterBar in project
2. Click on Filters(Adapt Filters) to open adapt filters dialog
3. If you use screen reader focus on Filter button, it doesn't read out the information
...

### Expected Behaviour

Screen readers should work as other button or filter component

### UI5 Web Components for React Version

2.5.0

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Operating System

MAC

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @JackieWei 

please add required information ([Contribution Guide](https://github.com/SAP/ui5-webcomponents-react/blob/main/CONTRIBUTING.md)), e.g. what announcements you would expect. `aria-role` or `role` is unnecessary here, as this is a button, `aria-level` is only meant for headings, so it would be wrong adding it here.
